src/measure_theory/vector_measure.lean

```diff
@@ -482,6 +482,9 @@ if hi : measurable_set i then
     { rw [Union_inter, if_pos (measurable_set.Union hf₁)] }
   end } else 0
 
+lemma restrict_not_measurable {i : set α} (hi : ¬ measurable_set i) : v.restrict i = 0 :=
+dif_neg hi
+
 lemma restrict_apply {i : set α} (hi : measurable_set i)
```

Comment:
This lemma (and likely many near it) should have `v` explicit, not implicit. That would remove the need for all the `@`s below, I think.